wallet/src/de/schildbach/wallet/data/BuyAndSellDashServicesModel.kt

```diff
@@ -20,7 +21,7 @@ data class BuyAndSellDashServicesModel(
     ) {
         LIQUID(org.dash.wallet.integration.liquid.R.string.liquid, org.dash.wallet.common.R.drawable.ic_liquid),
         UPHOLD(org.dash.wallet.integration.uphold.R.string.uphold_account, org.dash.wallet.common.R.drawable.ic_uphold),
-        COINBASE(org.dash.wallet.integration.coinbase_integration.R.string.coinbase, org.dash.wallet.common.R.drawable.ic_coinbase)
+        COINBASE(org.dash.wallet.integration.coinbase_integration.R.string.coinbase, R.drawable.ic_coinbase)
```

Comment:
I see no reason to put module-specific resources in the Common. If we do, it will grow to enormous proportions. Separation of resources is also a benefit in modular structure.